src/webgpu/api/validation/encoding/queries/resolveQuerySet.spec.ts

```diff
@@ -91,37 +85,35 @@ Tests that resolve query set with invalid destinationBuffer:
       usage: t.params.bufferUsage,
     });
 
-    const encoder = t.device.createCommandEncoder();
-    encoder.resolveQuerySet(querySet, 0, kQueryCount, destination, 0);
-
-    t.expectValidationError(() => {
-      encoder.finish();
-    }, t.params.bufferUsage !== GPUConst.BufferUsage.QUERY_RESOLVE);
+    const encoder = t.createEncoder('non-pass');
+    encoder.encoder.resolveQuerySet(querySet, 0, kQueryCount, destination, 0);
+    encoder.validateFinish(t.params.bufferUsage === GPUConst.BufferUsage.QUERY_RESOLVE);
   });
 
 g.test('resolveQuerySet,destination_offset')
   .desc(
     `
 Tests that resolve query set with invalid destinationOffset:
-- destinationOffset is not a multiple of 8
+- destinationOffset is not a multiple of 256
 - The size of destinationBuffer - destinationOffset < queryCount * 8
 - destinationOffset out of range
   `
   )
-  .paramsSubcasesOnly(u => u.combine('destinationOffset', [0, 6, 8, 16]))
+  .paramsSubcasesOnly(u => u.combine('destinationOffset', [0, 8, 256, 512, 768]))
```

Comment:
nit: test a "minimally unaligned" value, i.e. a multiple of the largest power of 2 less than 256:
```suggestion
  .paramsSubcasesOnly(u => u.combine('destinationOffset', [0, 128, 256, 384, 512, 768]))
```